web_disable_export_group/static/src/js/list_controller.esm.js

```diff
@@ -1,20 +1,23 @@
 /** @odoo-module **/
 /* Copyright 2018 Tecnativa - David Vidal
    License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
-import {ListController} from "@web/views/list/list_controller";
-import {onWillStart} from "@odoo/owl";
+import {archParseBoolean} from "@web/views/utils";
+import {exportAllItem} from "@web/views/list/export_all/export_all";
 import {patch} from "@web/core/utils/patch";
 
-patch(ListController.prototype, "disable_export_group", {
-    setup() {
-        this._super(...arguments);
-        onWillStart(async () => {
-            this.isExportEnable = await this.userService.hasGroup(
-                "base.group_allow_export"
-            );
-            this.isExportXlsEnable = await this.userService.hasGroup(
+patch(exportAllItem, {
+    async isDisplayed(env) {
+        const ExportEnable = super.isDisplayed(...arguments);
```

Comment:
The `super` function is async, so you must call it with await
```suggestion
        const ExportEnable = await super.isDisplayed(...arguments);
```

Reply:
@chienandalu 
done